src/components/cylc/Drawer.vue

```diff
@@ -36,19 +36,22 @@
       </v-list-item>
       <v-subheader>Views</v-subheader>
       <v-list-item
-        v-for="(link, index) in viewLinks"
-        :key="index+link.text"
-        :to="link.to"
-        active-class="primary grey--text text--darken-3"
-        class="v-list-item"
+          v-for="(link, index) in viewLinks"
+          :key="index+link.text"
+          :to="link.to"
+          active-class="primary grey--text text--darken-3"
+          class="v-list-item"
       >
         <v-list-item-action>
           <v-icon>{{ link.icon }}</v-icon>
         </v-list-item-action>
         <v-list-item-title
-          v-text="link.text"
+            v-text="link.text"
```

Comment:
Hm, I've added spaces to a few lines here without actually changing anything. Let me fix these.